sdk/loadtestservice/loadtestservice/package.json

```
@@ -0,0 +1,115 @@
+{
+  "name": "@azure-rest/loadtestservice",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0-beta.1",
+  "description": "",
+  "keywords": [
+    "node",
+    "azure",
+    "cloud",
+    "typescript",
+    "browser",
+    "isomorphic"
+  ],
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/loadtestservice.d.ts",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/undefined/README.md",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/loadtestservice.d.ts",
+    "README.md",
+    "LICENSE",
+    "review/*"
+  ],
+  "//metadata": {
+    "constantPaths": [{
+      "path": "swagger/README.md",
+      "prefix": "package-version"
+    }]
+  },
+  "engines": {
+    "node": ">=12.0.0"
```

@joheredi, @xirzec I think the time has come to move to v14:
```suggestion
    "node": ">=14.0.0"
```

---

Yup, @qiaozha we should update codegen